modules/users/server/controllers/users.profile.server.controller.js

```diff
@@ -299,6 +302,12 @@ exports.update = function (req, res) {
     });
   }
 
+  // validate locale
+  // @TODO validation framework
+  if (req.body.locale && (typeof req.body.locale !== 'string' || !Object.keys(languages).includes(req.body.locale))) {
+    return res.status(400).end();
```

Comment:
Just to keep things consistent with the rest of the API, let's return:

```js
return res.status(400).send({
  message: errorService.getErrorMessageByKey('bad-request')
});
```

Reply:
2f809c6